jha/src/views/enrolment/ContactInfoPage.vue

```diff
@@ -268,31 +287,40 @@ export default {
     this.mailPostalCode = this.$store.state.enrolmentModule.mailPostalCode;
     this.isMailSame = this.$store.state.enrolmentModule.isMailSame;
     this.phone = this.$store.state.enrolmentModule.phone;
+
+    if (this.onlyFPC) {
+      this.isMailSame = false;
+    }
   },
   validations() {
     let validations = {
-      resAddressLine1: {
-        required,
-        specialCharacterValidator,
-      },
-      resAddressLine2: {
-        specialCharacterValidator,
-      },
-      resAddressLine3: {
-        specialCharacterValidator,
-      },
-      resCity: {
-        required,
-        specialCharacterValidator,
-      },
-      resPostalCode: {
-        required,
-        bcPostalCodeValidator,
-      },
       phone: {
         phoneValidator,
       },
     };
+    if (!this.onlyFPC) {
+      let resValidations = {
```

Comment:
This should be a `const` since it's only being assigned an initial value.

Reply:
changed